examples/golang/verify.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -e
+
+export NAME=golang
+export UID
+export MANUAL=true
+
+# shellcheck source=examples/verify-common.sh
+. "$(dirname "${BASH_SOURCE[0]}")/../verify-common.sh"
+
+run_log "Compile the go plugin library"
+docker-compose -f docker-compose-go.yaml up --remove-orphans go_plugin_compile
+
+run_log "Start all of our containers"
+bring_up_example
+
+run_log "Make a request handled by the Go plugin"
+responds_with_header \
+    "rsp-header-from-go: bar-test" \
+    "http://localhost:8080"
```

Comment:
```suggestion
    "http://localhost:${PORT_PROXY}"
```

ditto other instances in this file